### PR TITLE
Minor Corrections

### PR DESCRIPTION
**Pull request by Denton-L**
Changed references from chriseth to ethereum, where relevant.

Corrected inconsistent whitespace issues.

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "name": "Solidity",
   "description": "Realtime compiler and runtime",
   "devtools_page": "devtools.html",
-  "update_url": "https://chriseth.github.io/browser-solidity/",
+  "update_url": "https://ethereum.github.io/browser-solidity/",
```
Comment (axic) [inline manifest.json]: Can you include this in #17  instead?

File: README.md [hunk 2]
```diff
@@ -15,10 +15,10 @@ loaded via http. If you clone / download the repository, use the
 
 Many dependencies are only provided via npm:
 
-    npm install    # fetch dependencies
+	npm install    # fetch dependencies
```
Comment (axic) [inline README.md]: Can you include this in #16 instead?

File: manifest.json (hunk 1 above)
Comment (Denton-L) [inline manifest.json]: I think the change was included in both pull requests.

**Comment by Denton-L** [comment]
Would you like me to move the changes from this PR into the other ones or is duplicating them fine?

**Comment by axic** [comment]
I think we should make a decision on the styling of the Javascript files before making changes to them. (Making big whitespace/styling changes also make merging the PRs against them harder.)

I would suggest `standard` or `semistandard` to be aligned with the majority of JS projects.

File: README.md (hunk 2 above)
Comment (Denton-L) [inline README.md]: This change is also duplicated in both PRs.

**Comment by Denton-L** [comment]
Are these changes alright, then? Instead of dictating whether to use spaces or tabs or whatever, I just made the formatting consistent between each individual file and removed trailing spaces.

**Comment by chriseth** [comment]
Please rebase.

**Comment by Denton-L** [comment]
Rebased changes on top of master.

**Pull request merged**